src/edit_view.rs

```diff
@@ -402,16 +358,30 @@ impl EditView {
                 match name.as_ref() {
                     "font_size" => {
                         if let Some(font_size) = value.as_u64() {
+                            let pango_ctx = self.da.get_pango_context().unwrap_or_else(|| {
```

Comment:
Is there a reason we don't just use `expect` here?

Reply:
Yes, see https://rust-lang.github.io/rust-clippy/master/index.html#expect_fun_call